src/js/components/Playlists/EmptyPlaylist.react.js

```diff
@@ -1,25 +0,0 @@
-import React, { PureComponent } from 'react';
-import FullViewMessage from '../Shared/FullViewMessage.react';
-
-/*
-|--------------------------------------------------------------------------
-| EmptyPlaylist
-|--------------------------------------------------------------------------
-*/
-
-export default class EmptyPlaylist extends PureComponent {
-
-    constructor(props) {
-
-        super(props);
-    }
-
-    render() {
-        return (
-            <FullViewMessage>
-                <p>Empty playlist</p>
-                <p className='sub-message'>You can add tracks from the library view</p>
-            </FullViewMessage>
-        );
-    }
-}
```

Comment:
Can we do same thing we did for queues: make container own either empty playlist or non-empty one? Naming for this would be hard though...